doc/sketch.rst

```diff
@@ -25,17 +25,38 @@ combining them using boolean operations.
        cq.Sketch()
        .trapezoid(4,3,90)
        .vertices()
-       .circle(.5,mode='s')
+       .circle(.5, mode='s')
        .reset()
        .vertices()
        .fillet(.25)
        .reset()
-       .rarray(.6,1,5,1).slot(1.5,0.4,mode='s',angle=90)
+       .rarray(.6,1,5,1).slot(1.5,0.4, mode='s', angle=90)
     )
 
 Note that selectors are implemented, but selection has to be explicitly reset. Sketch
 class does not implement history and all modifications happen in-place.
 
+Modes
+^^^^^
+
+Every operation from the face API accepts a mode parameter to defines how to combine the created object with existing ones. It can be fused (``mode='a'``), cut (``mode='s'``), intersected (``mode='i'``) or just stored for construction (``mode='c'``). In the last case, it is mandatory to specify a ``tag`` in order to be able to refer to the object later on. By default faces are fused together. Note the usage of the subtractive and additive modes in the example above. The additional two are demonstrated below.
```

Comment:
```suggestion
Every operation from the face API accepts a mode parameter to define how to combine the created object with existing ones. It can be fused (``mode='a'``), cut (``mode='s'``), intersected (``mode='i'``) or just stored for construction (``mode='c'``). In the last case, it is mandatory to specify a ``tag`` in order to be able to refer to the object later on. By default faces are fused together. Note the usage of the subtractive and additive modes in the example above. The additional two are demonstrated below.
```